packages/migrations/src/addDappnodePeerToIpfsNodes.ts

```diff
@@ -0,0 +1,26 @@
+import { logs } from "@dappnode/logger";
+
+//Adds Dappnode IPFS peers to the local IPFS node for more stability when fetching IPFS content
+export async function addDappnodePeerToIpfsNodes() {
+  const nodesIps = [
+    "167.86.114.131", //local node
+    "65.109.51.31", //production node
+  ];
+
+  const ipfsAlias = "ipfs.dappnode";
+  const dappnodeIpfsPeer = (ipfsNodeIp: string) =>
+    `/ip4/${ipfsNodeIp}/tcp/4001/p2p/QmfB6dT5zxUq1BXiXisgcZKYkvjywdDYBK5keRaqDKH633`;
+
+  // forEach node adding
+  nodesIps.forEach(async (node) => {
```

Comment:
Using forEach with asynchronous operations can lead to unexpected behaviors and difficulties in managing control flow. Please use other control flow like for const of or Promise.all